demo/demo-styles.css

```diff
@@ -0,0 +1,20 @@
+*{
+	transition: ease-in-out 150ms;
```

Comment:
Do you think it's prudent use a `* {}` in a lib? if I am an user, maybe for my grid would be nice to have this styles, but for the rest of the project not... 🤔

Reply:
Nope, this style is used for demo purpose only, that will animate our grid on our demo when the user resizes the device screen